app/assets/stylesheets/ama_layout/layout_components/banner.scss

```diff
@@ -0,0 +1,29 @@
+.banner{
+	position: relative; /* This is needed for the positioning of the close icon on the blue banners*/
+
+	&--blue{
+		background: $brand-blue-insurance;
+		padding: 20px;
+	}
+
+	&--red{
+		background: $brand-red;
+    padding: 42px 20px;
```

Comment:
does this padding need to be more than it is everywhere else? or can we just use padding: $base-padding instead?